numpy/add_newdocs.py

```diff
@@ -1453,7 +1453,7 @@ def luf(lamdaexpr, *args, **kwargs):
         When True, yield `x`, otherwise yield `y`.
     x, y : array_like, optional
         Values from which to choose. `x` and `y` need to have the same
-        shape as `condition`.
+        shape as `condition`, or be broadcastable to that shape.
```

Comment:
This isn't really correct either, although it's definitely closer to correct.

`(3,1)` and `(1, 3)` are not broadcastable to `()`, yet `np.where(True, np.zeros((3, 1)), np.zeros((1, 3)))` is valid